gigascience/gigadb-website
# Fix outstanding issues in change password page

# Pull request for issue: #1550

This is a pull request for the following functionalities:

* Fix layout and a11y issues in http://gigadb.gigasciencejournal.com:9170/user/changePassword page

## How to test?

* Navigate to http://gigadb.gigasciencejournal.com:9170/user/view_profile
* Verify that layout looks ok
* Run playwright tests. they all should pass. If running all tests gives error or takes too long, you can comment all paths except user/changePassword in playwright/tests/accessibility.spec.js
* Run WAVE audit, there should be no "errors" reported

## How have functionalities been implemented?

* Used component for text inputs, which are accessible out of the box
* Added accessibility features to checkboxes
* Fixed layout issues (button alignment)

## Any issues with implementation?

None

## Any changes to automated tests?

None

## Any changes to documentation?

None

## Any technical debt repayment?

None

## Any improvements to CI/CD pipeline?

None
